DependencyInjection/Compiler/LoggerChannelPass.php

```diff
@@ -112,4 +112,22 @@ protected function createLogger($channel, $loggerId, ContainerBuilder $container
             $this->channels[] = $channel;
         }
     }
+
+    /**
+     * Creates a copy of a reference and alters the service ID.
+     *
+     * @param Reference $reference
+     * @param string    $serviceId
+     *
+     * @return Reference
+     */
+    private function changeReference(Reference $reference, $serviceId)
+    {
+        if (method_exists($reference, 'isStrict')) {
+            // Stay compatible with Symfony 2
+            return new Reference($serviceId, $reference->getInvalidBehavior(), $reference->isStrict());
```

Comment:
`$reference->isStrict(false)` is probably the way to go. Ignoring the strict flag might cause problems with bundles that still rely on the deprecated behavior.